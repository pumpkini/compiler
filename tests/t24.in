void main {
	int a;
	string b;
	string c;
	a = 2;
	b = c = "string test";
}