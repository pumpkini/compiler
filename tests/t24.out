void
T_ID main
{
int
T_ID a
;
string
T_ID b
;
string
T_ID c
;
T_ID a
=
T_INTLITERAL 2
;
T_ID b
=
T_ID c
=
T_STRINGLITERAL "string test"
;
}